.changeset/clean-chairs-unite.md

```diff
@@ -0,0 +1,16 @@
+---
+'@commercetools-frontend/sdk': minor
+---
+
+The SDK request payload (body) now allows to pass something other than a `string` or `Json` objects.
```

Comment:
Nit: to me it's JSON. Also if the type name is different. I think in our docs it's always all capital letters.